servicebroker_node/config/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+	"port":8080, 
+	
+	"service_id": "autoscaler-guid",
```

Comment:
we extracted the configurable items in catalog here, so that it is easy to edit the key information of catalog with key-value format instead of taking care of the whole catalog JSON format